desktop/libs/notebook/src/notebook/api.py

```diff
@@ -405,7 +405,7 @@ def cancel_statement(request):
   response = {'status': -1}
 
   notebook = json.loads(request.POST.get('notebook', '{}'))
-  snippet = None
+  snippet = json.loads(request.POST.get('snippet', None))
```

Comment:
Nit: No need to provide `None` because `.get()` default to None if `snippet` is not found.

Reply:
Since you are expecting dict, so if required it should be defaulted to `{}` like in notebook.

Reply:
with the existing code line, we need `None` as we are checking the conditions with `not snippet`.

Reply:
Yes, I think we can use that